main/src/ca/uwaterloo/flix/language/ast/Name.scala

```diff
@@ -160,45 +160,13 @@ object Name {
     * @param namespace    the namespace
     * @param ident        the identifier.
     * @param loc          the source location of the qualified name.
-    * @param trailingDot  `true` if the qualified name ends with a dot (which implies this name is incomplete)
     */
-  case class QName(namespace: NName, ident: Ident, loc: SourceLocation, trailingDot: Boolean = false) {
```

Comment:
Lets a sentence to the comment that explain that the ident may be the empty string if there is a trailing dot.

Actually one or two examples would be good too.